test-network-function/operator/suite.go

```diff
@@ -71,26 +70,28 @@ var _ = ginkgo.Describe(testSpecName, func() {
 	}
 })
 
-func getConfig() []config.Operator {
+func getConfig() ([]config.CertifiedOperatorRequestInfo, []config.Operator) {
 	conf := config.GetConfigInstance()
-	return conf.Operators
+	operatorsToQuery := conf.CertifiedOperatorInfo
+	operatorsInTest := conf.Operators
+	return operatorsToQuery, operatorsInTest
 }
 
 func itRunsTestsOnOperator() {
-	operatorsInTest = getConfig()
-	gomega.Expect(err).To(gomega.BeNil())
-	gomega.Expect(operatorsInTest).ToNot(gomega.BeNil())
+	operatorsToQuery, operatorsInTest := getConfig()
+	gomega.Expect(operatorsToQuery).ToNot(gomega.BeNil())
 	certAPIClient := api.NewHTTPClient()
+	for _, certified := range operatorsToQuery {
+		ginkgo.It(fmt.Sprintf("test for Operator Certification Status of %s/%s", certified.Organization, certified.Name), func() {
```

Comment:
Similar to the other comment.

Reply:
It bugs me that the `container` and `operator` specs are structured so differently that there isn't an obvious `When` for this in the operator spec. Standardising them is for another time, though.
```
ginkgo.It(fmt.Sprintf("should eventually be verified as certified (operator %s/%s)", certified.Organization, certified.Name), ...
```